include/aspect/geometry_model/chunk.h

```diff
@@ -108,21 +108,66 @@ namespace aspect
         virtual
         Point<dim> representative_point(const double depth) const;
 
+        /**
+               * Return the longitude at the western edge of the chunk
```

Comment:
This looks really weird. Is this a mixture of tabs and spaces or something? Why does astyle not fix this?